examples/e2e-tests/elm/Generated/Api.elm

```diff
@@ -76,20 +79,60 @@ getIp =
         , body =
             Http.emptyBody
         , expect =
-            Http.expectStringResponse
+            Http.expectStringResponse toMsg
                 (\res ->
-                    Result.mapError Json.Decode.errorToString
-                        (Result.map
-                            (\body_ -> { url = res.url, status = res.status, headers = res.headers, body = body_ })
-                            (decodeString decodeResponse res.body)))
+                    case res of
+                        Http.BadUrl_ url -> Err (Nothing, Http.BadUrl url)
+                        Http.Timeout_ -> Err (Nothing, Http.Timeout)
+                        Http.NetworkError_ -> Err (Nothing, Http.NetworkError)
+                        Http.BadStatus_ metadata body_ -> Err (Just (metadata, body_), Http.BadStatus metadata.statusCode)
+                        Http.GoodStatus_ metadata body_ ->
+                            (decodeString decodeResponse body_)
+                                |> Result.mapError Json.Decode.errorToString
+                                |> Result.mapError Http.BadBody
+                                |> Result.mapError (Tuple.pair (Just (metadata, body_))))
         , timeout =
             Nothing
-        , withCredentials =
-            False
+        , tracker =
+            Nothing
         }
 
-getStatus204 : Http.Request (Http.Response (NoContent))
-getStatus204 =
+
+getIpSimulated : (Result (Maybe (Http.Metadata, String), Http.Error) (Response) -> msg) -> ProgramTest.SimulatedEffect msg
```

Comment:
I think we want this function to take a `String` from https://github.com/NoRedInk/servant-elm/pull/10 so that we can use that as the the `body_`

Reply:
(We talk IRL about how this is not necessary)